build/docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+version: '3.7'
```

Comment:
please don't name the directory build -> either name it differently like `local-telemetry` or smt more descriptive